docs/faq.rst

```diff
@@ -0,0 +1,102 @@
+.. _faq:
+
+Frequently asked questions
+==========================
+
+General fairness questions
+--------------------------
+
+Where can I learn more about fairness in machine learning?
+    In our :ref:`user guide <user_guide>` we provide links to the papers describing
+    Fairlearn's algorithms.
+    For a higher level view of the subject, please see the :ref:`further reading <further_reading>`
+    section.
+
+Why not just ignore the sensitive features?
+    Because ignoring the sensitive feature usually doesn't work.
+
+    To elaborate, information is often redundantly encoded in datasets, and machine learning
+    algorithms *will* ferret out these links (it is what they are designed to do).
+    For example, in the US, the ZIP code where a person lives is well correlated with their
+    race.
+    Even if the model is not provided with race as a feature, the model will pick up on it
+    implicitly via the ZIP code (and other features).
+    Worse, without having the race available in the dataset, it is not possible to determine
+    whether any use of ZIP code was as a proxy for race, or for some other reason.
+
+The model is unfair because the data are biased. Isn't it better to get better data?
+    Yes, but gathering unbiased data might not always be practical.
```

Comment:
Our fairness styleguide (PR) says that there are many possible reasons for why a model's predictions are unfair, not just bias.

I don't know whether we want to claim that gather unbiased data is possible at all actually. We can try to gather *less biased* data, but that in itself may introduce new biases, which is part of why we're focusing on harms instead.

I'm also not sure we want to talk about "redistributing harms" as opposed to mitigating harms. You can argue that by improving the accuracy for group A by 30% and worsening it for group B by 1% you did "redistribute", but I think it makes it seem like whatever you gain in some group you have to lose in another, and that's just not generally applicable. The package's goal is to help in mitigating unfairness.

Reply:
I've rephrased a bit. This is somewhere where @MiroDudik 's input would be very valuable - I was paraphrasing his response to the last question at Build here.